doc/classes/DisplayServer.xml

```diff
@@ -120,7 +120,7 @@
 			<description>
 				Displays OS native dialog for selecting files or directories in the file system.
 				Callbacks have the following arguments: [code]status: bool, selected_paths: PackedStringArray, selected_filter_index: int[/code].
-				[b]Note:[/b] This method is implemented if the display server has the [constant FEATURE_NATIVE_DIALOG] feature, i.e. Linux (X11), Windows, and macOS.
+				[b]Note:[/b] This method is implemented if the display server has the [constant FEATURE_NATIVE_DIALOG] feature, i.e. Linux (X11 and Wayland), Windows, and macOS.
```

Comment:
Perhaps? It's kind of weird start with "display server" and then list the platforms with no connection.
```suggestion
				[b]Note:[/b] This method is implemented if the display server has the [constant FEATURE_NATIVE_DIALOG] feature. Supported platforms include Linux (X11 and Wayland), Windows, and macOS.
```

Reply:
I'm fine either way. It's just that i.e. ("That is") never really fit here.